### PR TITLE
Upgrade project dependencies

### PR DESCRIPTION
**Pull request by kmcnellis**
Re-ran the generator, and updated all the dependencies.
Now we use sass instead of less, and sockets, babel, and bluebird are included.

### CODE REVIEW

**Comment by agundy** [comment]
Failed test: 
`Running "mochaTest:unit" (mochaTest) task

>> Mocha exploded!

>> Error: Cannot find module 'md5'`

I think you may need to update the import?

**Comment by agundy** [comment]
Npm install still has deprecated dependency warnings. Maybe see if we could get them good?

**Comment by agundy** [comment]
remove the yeoman.png please!

File: client/index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h5 style="margin-left:10px;">
             m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
                   })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-      ga('create', 'UA-35949221-4', 'auto');
```
Comment (agundy) [inline client/index.html]: should add the Google anylitics code back in

File: server/auth/local/index.js [hunk 2]
```diff
@@ -1,20 +1,24 @@
...
+      return res.status(401).json(error);
+    }
+    if (!user) {
+      return res.status(404).json({message: 'Something went wrong, please try again.'});
+    }
 
-    var token = auth.signToken(user._id, user.role);
-    res.json({token: token});
+    var token = signToken(user._id, user.role);
+    res.json({ token });
```
Comment (agundy) [inline server/auth/local/index.js]: is this something you changed? or is this generated?

**Comment by agundy** [comment]
address the concerns and then looks good to go

**Comment by agundy** [comment]
Before this pushes I will also do a db backup

File: server/auth/local/index.js (hunk 2 above)
Comment (kmcnellis) [inline server/auth/local/index.js]: It was generated

**Comment by kmcnellis** [comment]
Ok, I'm going to squash it

**Pull request merged**